react/react2/week1/src/components/list/List.jsx

```diff
@@ -0,0 +1,19 @@
+import { useContext } from "react";
+import {AppContext} from "../../App";
+
+import Item from '../item/Item.jsx'
+
+import './List.css';
+
+const List = () => {
+  const { results } = useContext(AppContext);
+  console.log(results);
+
+  return (
+    <>
+      {results.length?  results.map((user) => <Item key={user.id}>{user.login}</Item>) : null}
```

Comment:
might want to wrap the unordered list (<ul>) tag around the map function, so that the list items are in one list and not several

Reply:
Thanks for review. I have made changes